taxonomies/categories.txt

```diff
@@ -48613,10 +48613,10 @@ fr:Yaourts brassés, Yaourt brassé
 nl:Roeryoghurts
 #wikidata:en:
 
-<ru:Йогурт
+<en:Yogurts
 ru:Биойогурт, Биойогурты
 
-<ru:Йогурт
+<en:Yogurts
```

Comment:
I think this category is a synonyme of "Fruit yogurts"

Reply:
OK